Cargo.toml

```diff
@@ -29,6 +29,30 @@ exclude = [
 [lints.rust]
 unexpected_cfgs = { level = "warn", check-cfg = ['cfg(fuzzing)'] }
 
+[workspace.dependencies]
+uuid = { version = "1.10", default-features = false }
+tracing = { version = "0.1",  default-features = false }
+bitflags = "2.6"
+rand = "0.8"
+cfg-if = "1"
+time = { version = "0.3", default-features = false }
+sha1 = { version = "0.10", default-features = false }
+num-derive = "0.4"
+num-traits = { version = "0.2", default-features = false }
+picky = { version = "7.0.0-rc.9", default-features = false }
+picky-asn1-der = "0.5"
+picky-asn1-x509 = "0.13"
+# `rt-multi-thread` feature is required for `tokio::task::block_in_place` function.
+tokio = { version = "1.40", features = ["time", "rt", "rt-multi-thread"] }
+ffi-types = { path = "crates/ffi-types" }
+winscard = { path = "crates/winscard" }
+rsa = { version = "0.9.6", default-features = false }
+windows-sys = { version = "0.59" }
+base64 = "0.22"
+whoami = "1.5"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+proptest = "1.5"
```

Comment:
question: Is it possible to keep the `default-features = false` and `features = […]` in the individual Cargo.toml? This is typically stuff that we want to determine at the crate level, not the workspace level. Spreading the definitions across many files is also not very good for the readability I think. I would prefer we only keep version numbers in the workspace manifest.

Reply:
Yes, it's possible.
I didn't think about it, but now it sounds reasonable. I improved the `Cargo.toml`